### ISSUE TITLE
All options in the settings menu are not visible after activating the theme

### ISSUE THREAD

**Issue by raviece2k10** [Nilahn/pine_forest_berry#4]
In the setting menu, few options at the last of the page are not visible. PFB snaps.

**In this theme:**
![image](https://github.com/Nilahn/pine_forest_berry/assets/97590564/eb6a343d-898e-4d42-81c1-3bc07b112d92)
**Minimal/Other Theme:**
![image](https://github.com/Nilahn/pine_forest_berry/assets/97590564/0f994f85-733f-41f6-b08f-a7c14d2764f0)

**In this theme:**
![image](https://github.com/Nilahn/pine_forest_berry/assets/97590564/62490392-41d0-426e-b9d6-345f9ba8ba94)
**Minimal/Other Theme:**
![image](https://github.com/Nilahn/pine_forest_berry/assets/97590564/19c0406f-ae3d-44bf-a576-2826ce8a4e01)

**In this theme:**
![image](https://github.com/Nilahn/pine_forest_berry/assets/97590564/7fbe8d54-b3ab-4925-91cc-ee93aeef2aec)
**Minimal/Other Theme:**
![image](https://github.com/Nilahn/pine_forest_berry/assets/97590564/00d37d5e-6f18-4c72-a7c3-79b805d38a9a)

**In this theme:**
![image](https://github.com/Nilahn/pine_forest_berry/assets/97590564/e6420a48-6bee-44d6-ab97-014b493f5977)
**Minimal/Other Theme:**
![image](https://github.com/Nilahn/pine_forest_berry/assets/97590564/2851fe78-a9fe-4bbe-aea7-506e55595c1b)

> It is my favourite theme among all Obsidian themes, don't want it to leave for just one reason. Hopefully it will be fixed soon. Thanks in advance.

**Comment by Nilahn** [comment]
Yes! I've noticed that and wanted to fix that for a while. Thanks for the kick in the butt. Should be working now!
